src/App.js

```diff
@@ -1,23 +1,24 @@
 import React, { useState } from 'react';
 import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
 import './App.css';
-import Button from './components/Button';
+import TokenList from './Components/List/tokenList';
```

Comment:
This is causing an error in the build. Make sure you change `Components/` with `components` (lowercase) before merging.

Reply:
Hello, Thank you very much for the comments, they help us to grow a lot. I want to say that when I try to rename "Components" it doesn't allow me
![image](https://user-images.githubusercontent.com/54757879/96369624-c237e200-1120-11eb-93d8-6c743790e941.png)
If I leave it in capital letters, the application runs normally
![image](https://user-images.githubusercontent.com/54757879/96369670-fe6b4280-1120-11eb-838d-757071e6f1fc.png)
and if I put it in lowercase the compilation error occurs 
![image](https://user-images.githubusercontent.com/54757879/96369701-2ce91d80-1121-11eb-88f8-066bc69613a9.png)

Reply:
@Lizhtml @AndrewEspejo, it seems that error is gone now and the build is passing! 🤷 

Let's leave this as is then,

Thank you!